daemon/info.go

```diff
@@ -75,7 +75,7 @@ func (daemon *Daemon) SystemInfo() (*types.Info, error) {
 		DriverStatus:       daemon.layerStore.DriverStatus(),
 		Plugins:            daemon.showPluginsInfo(),
 		IPv4Forwarding:     !sysInfo.IPv4ForwardingDisabled,
-		BridgeNfIptables:   !sysInfo.BridgeNfCallIptablesDisabled,
+		BridgeNfIptables:   !sysInfo.BridgeNfCallIPtablesDisabled,
```

Comment:
@aaronlehmann, I do think that BridgeNfIptables should be updated to be consistent too. That would be a change to engine-api/types/types.go though. Will that break API things?